FRW/Services/Manager/WalletManager.swift

```diff
@@ -885,11 +893,17 @@ extension WalletManager: FlowSigner {
             try await findFlowAccount()
         }
         if userSecretSign() {
-            if let userId = walletInfo?.id, let data = try WallectSecureEnclave.Store.fetch(by: userId) {
-                let sec = try WallectSecureEnclave(privateKey: data)
-                let signature = try sec.sign(data: signableData)
+            if let userId = walletInfo?.id {
+                let secureKey = try SecureEnclaveKey.wallet(id: userId)
+                let signature = try secureKey.sign(data: signableData, hashAlgo: .SHA2_256)
                 return signature
             }
+            
+//            if let userId = walletInfo?.id, let data = try WallectSecureEnclave.Store.fetch(by: userId) {
+//                let sec = try WallectSecureEnclave(privateKey: data)
+//                let signature = try sec.sign(data: signableData)
+//                return signature
+//            }
```

Comment:
Yet another case of commented code without a description. This seems very similar to the one at like 864, so in case it will be used in the future, better to move to a separate private method.